client/src/test/suite/extension.test.ts

```diff
@@ -60,6 +64,19 @@ const resetTestData = async (): Promise<void> => {
   fs.copySync(ORIGIN_DATA_DIR, TEST_DATA_DIR)
 }
 
+// TODO Where does this go?
+const stubRepo = Substitute.for<Repository>()
+
+const getRepo = (): Repository => {
+  return stubRepo
+}
+
+const reporter = (expectedMessage: string) => (_: string) => {
```

Comment:
Might be a better name to indicate an assertion is being made inside the tests, up to you.
```suggestion
const assertReported = (expectedMessage: string) => (_: string) => {
```